src/theme/components/breadcrumb.ts

```diff
@@ -0,0 +1,75 @@
+// Variant styling
+const booksAndMore = {
+  bg: "section.books-and-more.secondary",
+};
+const locations = {
+  bg: "section.locations.secondary",
+};
+const research = {
+  bg: "section.research.secondary",
+};
+const whatsOn = {
+  bg: "section.whats-on.secondary",
+};
+
+const Breadcrumb = {
+  baseStyle: {
+    bg: "ui.black",
+    color: "ui.white",
+    fontSize: "-1",
+    fontWeight: "regular",
+    paddingBottom: "xs",
+    paddingTop: "xs",
+    ".chakra-breadcrumb__list": {
```

Comment:
You can target the `ol` element directly instead of the class name.

Reply:
Yes, that works well here.  Thanks.  I thought I could do something similar for the `.chakra-breadcrumb__link`, but that is used on both `a` and `span`.